tribe-ext-list-venues-organizers-shortcodes.php

```diff
@@ -139,7 +139,8 @@ protected function format() {
 
 				// count upcoming events
 				$args = [
-					'start_date' => date( 'Y-m-d H:i:s' ),
+					'start_date'     => date( 'Y-m-d H:i:s' ),
+					'posts_per_page' => -1, // needed in order to get an accurate count
```

Comment:
Querying for _all_ upcoming events doesn't strike me as a great idea from a performance perspective. 

Can we look at ways of mitigating this, such requesting 1 post and asking `tribe_get_events()` to return the complete query object and then inspecting the `found_posts` property to determine how many exist in total?

Reply:
@barryhughes great idea! Updated. OK to merge now?